src/dialog/Dialog.ts

```diff
@@ -94,15 +94,13 @@ export default class Dialog extends DialogBase<DialogProperties> {
 			}, [
 				v('div', {
 					classes: this.classes(css.title),
-					id: this._titleId,
 					key: 'title'
 				}, [
-					title,
+					v('div', { id: this._titleId }, [ title ]),
 					closeable ? v('button', {
 						classes: this.classes(css.close),
```

Comment:
What styling would need to be in a fixed class? With no theme (base or otherwise) the close button is still visible in the titlebar and clickable, it just has no styling.